clientv3/integration/dial_test.go

```diff
@@ -23,9 +23,51 @@ import (
 	pb "github.com/coreos/etcd/etcdserver/etcdserverpb"
 	"github.com/coreos/etcd/integration"
 	"github.com/coreos/etcd/pkg/testutil"
+	"github.com/coreos/etcd/pkg/transport"
+
 	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+)
+
+var (
+	testTLSInfo = transport.TLSInfo{
+		KeyFile:        "../../integration/fixtures/server.key.insecure",
+		CertFile:       "../../integration/fixtures/server.crt",
+		TrustedCAFile:  "../../integration/fixtures/ca.crt",
+		ClientCertAuth: true,
+	}
+
+	testTLSInfoBad = transport.TLSInfo{
+		KeyFile:        "../../integration/fixtures-bad/server-key.pem",
+		CertFile:       "../../integration/fixtures-bad/server.pem",
+		TrustedCAFile:  "../../integration/fixtures-bad/etcd-root-ca.pem",
+		ClientCertAuth: true,
+	}
 )
 
+func TestDialTLS(t *testing.T) {
+	defer testutil.AfterTest(t)
+	clus := integration.NewClusterV3(t, &integration.ClusterConfig{Size: 1, PeerTLS: &testTLSInfo, ClientTLS: &testTLSInfo})
+	defer clus.Terminate(t)
+
+	c1, err := integration.NewClientV3(clus.Members[0])
```

Comment:
can this be left out? Seems like other tests should cover the success case already